src/strategies/strategies/nato-uniswap-v3/examples.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "strategy": {
+      "name": "nato-uniswap-v3",
+      "params": {
+        "tokenAddress": "0xd968196fa6977c4e58f2af5ac01c655ea8332d22",
+        "poolAddress": "0x02623e0e65A1D8537f6235512839E2f7b76C7A12",
+        "feeTier": 10000,
+        "positionManager": "0x03a520b32c04bf3beef7beb72e919cf822ed34f1",
+        "network": "base"
+      }
+    },
+    "addresses": ["0xdA1dbA3c1B1cdEAf1419a85bA5B454547bdA5662"],
+    "snapshot": "latest"
+  }
```

Comment:
`network` should be passed at this param https://github.com/snapshot-labs/score-api/blob/master/src/strategies/strategies/erc20-balance-of/examples.json#L12
check other strategies for examples

src/strategies/strategies/nato-uniswap-v3/index.ts